api/RTCIceTransport.json

```diff
@@ -243,56 +243,6 @@
           }
         }
       },
-      "gatheringstatechange_event": {
```

Comment:
Looks like the event is in Chromium, Gecko and WebKit source and `ongatheringstatechange` has support data:
https://github.com/mdn/browser-compat-data/blob/820c07582578aab9b125ef8923d3b11f3f108cc1/api/RTCIceTransport.json#L548-L596

Just copying that data would be a good start to show that this shouldn't be removed.